packages/plugins/rubygem-hammer_cli_foreman_templates/rubygem-hammer_cli_foreman_templates.spec

```diff
@@ -8,7 +8,7 @@
 %global hammer_confdir %{_root_sysconfdir}/hammer
 
 Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 0.1.0
+Version: 0.1.2
 Release: 3%{?foremandist}%{?dist}
```

Comment:
The release should be set back to 1. https://github.com/theforeman/foreman-packaging/blob/rpm/develop/bump_rpm.sh is supposed to take care of this. Did you use the script and did it fail to?

Reply:
Yes, I used the script, but I'm afraid this is my fault, because I didn't (?) actually release rpm version of 0.1.1.

So should I create another one PR with 0.1.1 gem before this one?

Reply:
Skipping versions is not a problem.